pb/edge_ctrl_pb/edge_ctrl.proto

```diff
@@ -54,11 +54,13 @@ message Session {
     repeated string urls = 4;
     Service service = 5;
     string id = 7;
+    string apiSessionId = 8;
 }
 
 message ApiSession {
     string token = 1;
     repeated string certFingerprints = 2;
+    string id = 3;
 }
```

Comment:
These allow sessions and API Sessions to easily be linked in the router's cache.